assets/css/main.css

```diff
@@ -130,6 +125,10 @@ button.destructive {
 #page-actions > li:hover > summary > i {
   transform: rotate(90deg);
 }
+nav details[role="list"] summary,
+nav li[role="list"] a {
+  display: block;
+}
```

Comment:
simplify markup for buttons in nav:
- from `<nav><div><button>`
- to `<nav><button>`